material-ui-superselectfield/material-ui-superselectfield.d.ts

```diff
@@ -0,0 +1,45 @@
+declare module "material-ui-superselectfield" {
+    import * as React from "react"
+
+    interface IMaterialUISuperSelectFieldProps {
+        name: string;
+        value: object[] | object | null;
+        multiple?: boolean;
+        key?: string;
+        onChange?: (...args: any[]) => void;
```

Comment:
I updated the functions to use rest parameters.  This allows for any number of (including 0) arguments to be passed into the function, to keep these typings as flexible as possible for the future.

https://stackoverflow.com/questions/12739149/typescript-type-signatures-for-functions-with-variable-argument-counts
https://www.typescriptlang.org/docs/handbook/functions.html